docs/docs/metrics.md

```diff
@@ -119,6 +119,16 @@ forwarded as gauges.
   the current Marathon instance since it became a leader.
 * `marathon.uptime.gauge.seconds` — uptime of the current Marathon
   instance.
+* `marathon.instances.overdue.gauge` - the number of overdue instances. Overdue 
+  instances are those that has been either launched or staged but hasn't 
+  become running within the `task_launch_timeout` or `task_launch_confirmation_timeout` 
+  respectively.
+* `marathon.instances.doomed.gauge` - the current number of instances for which 
```

Comment:
`unkillable` doesn't fit, since we don't respect the `killGraceTimePeriod` here. The task can simply take its time but we will still retry. `killing` doesn't fit since it's a valid Mesos status update. 
However, I'm not super happy with `doomed` either so I'm open to suggestions.